ci/tasks/test-get.sh

```diff
@@ -1,7 +1,29 @@
 #! /bin/sh
 
-set -ex
+set -e
 
-tree bucket.s3simple
+cd bucket.s3simple
+
+echo
+echo "All files downloaded from the bucket:"
+ls -R -1
+
+echo
+echo "Contents of snapshot file downloaded from the bucket:"
+echo
+cat snapshot
+
+# Read the snapshot file we found in the bucket and confirm that all the files are present.
+echo
+echo "Validating that the files in snapshot have been downloaded:"
```

Comment:
No more manual verification? The job will actually fail if the content does not match. Seems much better :)